.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: ci
+
+on:
+  push:
+    branches: [ master, qa ]
+  pull_request:
+    branches: [ master, qa ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '1.11.x']
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install
+      env:
+        GO111MODULE: on
+      run: |
+        go get .
+    - name: Test
+      shell: bash
+      run: |
+        go version
+        go test -v google.golang.org/appengine/...
+        go test -v -race google.golang.org/appengine/...
```

Comment:
I am not that familiar with go, why are you running the same tests twice?

Reply:
This test command I copied over from old travis test. Fair point, I think it's safe to just run it once with data race detection on.